initfiles/bash/etc/init.d/pid.sh

```diff
@@ -174,7 +174,7 @@ check_status() {
 checkSentinelFile() {
     FILEPATH="${runtime}/${compName}"
     if [[ -d ${FILEPATH} ]]; then
-       fileCheckOP=$(find ${FILEPATH} -name "*senti*")
+       fileCheckOP=$(find ${FILEPATH} -maxdepth 1 -name "*senti*")
```

Comment:
Looks good to me.
An ls -1 ${FILEPATH}/*senti* 2> /dev/null might have worked also.